tools/db/scim-emails/src/Work.hs

```diff
@@ -67,8 +82,14 @@ getEmailAndExternalId uid =
     cql :: PrepQuery R (Identity UserId) (Maybe Email, Maybe (Writetime Email), Maybe Email, Maybe UserSSOId)
     cql = "SELECT email, writetime(email), email_unvalidated, sso_id from user where id = ?"
 
-checkUser :: ClientState -> ClientState -> Logger -> (UserId, Maybe UserId, Maybe UTCTime) -> IO ()
-checkUser brig spar l (uid, mInvitedBy, mInvitedAt) = do
+getSCIMTeams :: ConduitM () [Identity TeamId] Client ()
+getSCIMTeams = paginateC cql (paramsP LocalQuorum () pageSize) x5
+  where
+    cql :: PrepQuery R () (Identity TeamId)
+    cql = "SELECT team from scim_external"
```

Comment:
I think the other table is called `scim_external_ids`, and we should extract users from both rather than extracting teams from just one.  Unless you can confirm that `scim_external_ids` is empty.

Reply:
`scim_external_ids` gives us users and not teams, I wanted to iterate over teams. But maybe it is also ok to just iterate over users which spar thinks are SCIM users?

Reply:
yes.